### ISSUE TITLE
CTL/ATL values in 2.0 twice as high than in 1.5 - recommendation for rest/competition wrong

### ISSUE THREAD

**Issue by grzeg1** [Runalyze/Runalyze#1287]
New values are twice as high as the old ones. Is that a fix or a bug? Also the suggested ranges for TSB (e.g. >10 for competition) seem not to be valid now.

**Comment by laufhannes** [comment]
RPE was used as additional factor in v1.5 which causes these differences:
```php
$Trimp = round($s/60 * self::TrimpFactor($avgHF) * $RPE / 10);
```

And I guess there is no globally valid suggestion for TSB. I'm not sure if we should keep that in any way.
in my opinion there is no globally valid suggestion for TSB.

**Comment by grzeg1** [comment]
If the recommendation is not valid any more that this is a bug because it needs to be removed.

**Comment by laufhannes** [comment]
Fixed with 7a28b04ecde3f88b2c764bd1948d0b77bfd2d520.
